sources_test.go

```diff
@@ -155,6 +155,40 @@ func TestListFilesGlob(t *testing.T) {
 	}
 }
 
+func TestFindFilesMultipleFiles(t *testing.T) {
+	t.Parallel()
+	dir := "testdata/multiple_files"
+	want := fmt.Sprintf("%s/1.txt\n%s/2.txt\n%s/3.tar.zip\n", dir, dir, dir)
+	got, err := ListFiles(dir).String()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if got != want {
```

Comment:
I try to use `cmp.Equal` and `cmp.Diff` wherever possible in the tests, because the output is so much more helpful than simply comparing strings. See other tests for how to use these.